aspnet/SameSite/owin-samesite.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Work with SameSite cookies and the Open Web Interface for .NET (OWIN)
+author: rick-anderson
+description: Work with SameSite cookies and the Open Web Interface for .NET (OWIN)
+ms.author: riande
+ms.date: 12/6/2019
+uid: owin-samesite
+---
+
+# SameSite cookies and the Open Web Interface for .NET (OWIN)
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+SameSite is an [IETF](https://ietf.org/about/) draft designed to provide some protection against cross-site request forgery (CSRF) attacks. The [SameSite 2019 draft](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00):
+
+* Treats cookies as `SameSite=Lax` by default.
+* States cookies that explicitly assert `SameSite=None` in order to enable cross-site delivery should be marked as `Secure`.
+
+`Lax` works for most app cookies. Some forms of authentication like [OpenID Connect](https://openid.net/connect/) (OIDC) and [WS-Federation](https://auth0.com/docs/protocols/ws-fed) default to POST based redirects. The POST based redirects trigger the SameSite browser protections, so SameSite is disabled for these components. Most [OAuth](https://oauth.net/) logins are not affected due to differences in how the request flows.
```

Comment:
All other components do not set SameSite by default and use the clients default behavior (old or new).